antarest/study/business/allocation_management.py

```diff
@@ -139,7 +139,8 @@ def get_allocation_data(self, study: StudyInterface, area_id: str) -> Dict[str,
         if not allocation_data:
             raise AllocationDataNotFound(area_id)
 
-        return allocation_data.get("[allocation]", {})  # type: ignore
+        # allocation format can differ from the number of '[' (i.e. [[allocation]] or [allocation])
```

Comment:
well spotted, this is unbelievable ...

Actually, when the simulator reads the file, it seems to not even care about what's in the section name.
However when it writes to the file, it writes `[[allocation]]`